### ISSUE TITLE
Working Implementation ?

### ISSUE THREAD

**Issue by Pravesh001** [Aftaab99/OfflineSignatureVerification#2]
I’m not sure how to implement this project in working. I checked the help docs (Readme.md) and didn’t find any mentions.

**Comment by Aftaab99** [comment]
The project is working, there are steps in the README with commands to download the pre-trained model and dataset, and then start the web server for the demo. I'll write the commands here again in case your having trouble.

### Setting up locally

Download the dependencies(for conda users)

    conda env create -f projenv.yml

They can be installed using pip via

    pip install -r requirements.txt

Then download the pretrained model and dataset

    python DownloadData.py

After setting the environment and downloading the pre-trained model, start the web server using

    python main.py

The demo starts at port 5000. Links to upload and verification page are under the menu.


I didn't quite understand what you meant by 'in working' but if you want to use the pre-trained model for inference, you'll need to find a threshold value which gives the best accuracy and then have the model predict true or false if the Euclidean distance between the two feature vectors is greater or lesser than the threshold respectively. You can look into `Test.py` as to how to do all this.

**Comment by Aftaab99** [comment]
Closing this issue. Reopen it if you please, in case they are any mistakes in the implementation.

**Comment by Pravesh001** [comment]
Thanks for your response and help.
I'm clear now.
